actor/builtin/miner/miner_test.go

```diff
@@ -361,6 +361,103 @@ func TestMinerSubmitPoSt(t *testing.T) {
 	require.EqualError(res.ExecutionError, "submitted PoSt late, need to pay a fee")
 }
 
+func TestVerifyPIP(t *testing.T) {
+	tf.UnitTest(t)
+
+	ctx := context.Background()
+	st, vms := core.CreateStorages(ctx, t)
+
+	ancestors := th.RequireTipSetChain(t, 10)
+
+	origPid := th.RequireRandomPeerID(require.New(t))
+	minerAddr := createTestMiner(assert.New(t), st, vms, address.TestAddress, []byte("my public key"), origPid)
+
+	sectorId := uint64(1)
+	commD := th.MakeCommitment()
+
+	// add a sector
+	res, err := th.CreateAndApplyTestMessage(t, st, vms, minerAddr, 0, 3, "commitSector", ancestors, sectorId, commD, th.MakeCommitment(), th.MakeCommitment(), th.MakeRandomBytes(int(types.SealBytesLen)))
+	require.NoError(t, err)
+	require.NoError(t, res.ExecutionError)
+	require.Equal(t, uint8(0), res.Receipt.ExitCode)
+
+	runVerifyPIP := func(t *testing.T, bh uint64, commP []byte, sectorId uint64, proof []byte) (bool, string) {
+		res, err = th.CreateAndApplyTestMessage(t, st, vms, minerAddr, 0, bh, "verifyPIP", ancestors, commP, sectorId, proof)
+		require.NoError(t, err)
+		require.NoError(t, res.ExecutionError)
+		require.Equal(t, uint8(0), res.Receipt.ExitCode)
+
+		valid := parseAbiBoolean(res.Receipt.Return[0])
+		reason := string(res.Receipt.Return[1])
```

Comment:
Make assertions about the other two return values.

Reply:
All of the return values of the actor's method are actually being checked, but it is nonetheless very confusing:

The `uint8` return value from the actor method is checked in the assertion on `res.Receipt.ExitCode` above.

The `errror` returned from the actor method is checked in the assert on `err` above.